metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -105,6 +106,41 @@ class Constant:
     DATASET_WEB_URL = "datasetWebUrl"
 
 
+@dataclass
+class DataPlatformPair:
+    datahub_data_platform_name: str
+    powerbi_data_platform_name: str
+
+
+@dataclass
+class DataPlatformTable:
+    name: str
+    full_name: str
+    data_platform_pair: DataPlatformPair
+
+
+class SupportedDataPlatform(Enum):
+    POSTGRES_SQL = DataPlatformPair(
+        powerbi_data_platform_name="PostgreSQL", datahub_data_platform_name="postgres"
+    )
+
+    ORACLE = DataPlatformPair(
+        powerbi_data_platform_name="Oracle", datahub_data_platform_name="oracle"
+    )
+
+    SNOWFLAKE = DataPlatformPair(
+        powerbi_data_platform_name="Snowflake", datahub_data_platform_name="snowflake"
+    )
+
+    MS_SQL = DataPlatformPair(
```

Comment:
How about Redshift?

Reply:
It will automatically get supported once this PR get merged: https://github.com/datahub-project/datahub/pull/7562